src/utils/optionsdict.h

```diff
@@ -144,6 +160,9 @@ class OptionsDict : TypeDict<bool>,
   // Returns list of subdictionaries.
   std::vector<std::string> ListSubdicts() const;
 
+  // Adds alias dictionary.
+  void AddAliasDict(const OptionsDict* dict);
```

Comment:
This (trivial?) methods implementation oddly seems missing?

Reply:
It's there in `optionsdict.cc`.